entries/even.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<entry type="method" name="even" return="jQuery">
+  <title>.even()</title>
+  <signature>
+    <added>3.5</added>
+  </signature>
+  <desc>Reduce the set of matched elements to the even ones in the set.</desc>
```

Comment:
I know you mentioned it below but for people who don't read maybe `...in the set, numbered from zero.`

Reply:
Good point!